lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -227,6 +229,22 @@ sub pick_siblings_of_running {
     }
 }
 
+# 50 is the default (at the moment of writing :)
+# we want a 1 for prio 50, a 0.1 for prio 100
+# and 10 for prio 0 so that high prio jobs get quicker
+# a multi machine peer
+sub offset_from_prio {
+    my $priority = shift;
+    $priority -= OpenQA::Schema::JobGroupDefaults::PRIORITY;
```

Comment:
Looks like `OpenQA::Schema::JobGroupDefaults` is not imported.

Reply:
the tests still pass :)